bin/zold

```diff
@@ -125,6 +126,8 @@ Available options:"
     exit
   end
 
+  Zold::VersionUpgradeManager.run
```

Comment:
@botanicus rename it to `Upgrades`. See https://www.yegor256.com/2015/03/09/objects-end-with-er.html